i_favoriten_liste_implementieren

ImSeDanny opened the pull request:
A_Favoriten:
- Content von A_Garantie kopiert und entsprechend abgeändert --> auf die IDs der box_favoriten_view.xml geändert
- bons.setFavoritent()
- bons.deleteFavorieten()

C_Favoriten_Adapter:
- Klasse erstellt
- Content von C_Garantie_Adapter kopiert und entsprechend abgeändert

box_favoriten_content.xml:
- Content von box_garantie_content.xml kopiert und entsprechend abgeändert

box_favoriten_view.xml:
- Content von box_garantie_view.xml kopiert und entsprechend abgeändert --> IDs auf favoriten geändert

C_Bons_Adapter:
- Favoriten-Sterne mit primary Color getinted

ImSeDanny committed:
Favoriten Liste implementiert

A_Favoriten:
- Content von A_Garantie kopiert und entsprechend abgeändert --> auf die IDs der box_favoriten_view.xml geändert
- bons.setFavoritent()
- bons.deleteFavorieten()

C_Favoriten_Adapter:
- Klasse erstellt
- Content von C_Garantie_Adapter kopiert und entsprechend abgeändert

box_favoriten_content.xml:
- Content von box_garantie_content.xml kopiert und entsprechend abgeändert

box_favoriten_view.xml:
- Content von box_garantie_view.xml kopiert und entsprechend abgeändert --> IDs auf favoriten geändert

Sippl commented on app/src/main/res/layout/box_favoriten_view.xml:
```diff
@@ -0,0 +1,53 @@
+        android:layout_alignParentRight="true"
+        android:layout_margin="16dp"
+        android:layout_centerVertical="true"
+        android:src="@mipmap/ic_delete_black_24dp"
```
Hey Diddy, ich finde hier den "Delete" Button sehr irreführend (Auch verwechselbar mit Garantie Liste). Im Home Tab und Bon Tab ist immer ein Stern und plötzlich ein Mülleimer. Meiner Meinung wäre es sinnvoller anstatt dem Mülleimer einen Stern wie im bsp. im Home Tab zu verwenden. Könntest du dies bitte ändern?

ImSeDanny committed:
Sterne Bug gefixed

box_bons_view_layout.xml:
- Farbe des Sterns auf primaryColor getinted

Sippl commented on app/src/main/java/de/projektss17/bonpix/daten/C_Favoriten_Adapter.java:
```diff
@@ -0,0 +1,108 @@
+        public MyViewHolder(View view){
+            super(view);
+            icon = (ImageView) view.findViewById(R.id.favoriten_view_laden_bild);
+            favoriteShopName = (TextView) view.findViewById(R.id.favoriten_view_favoritenbeginn);
```
Hi Diddy :-), warum hast du hier dies "favoritenbeginn" und drunter "favoritenende" genannt? Kannst du dies bitte in der XML umbennen. Weil das eine ist ja favoriteshopname und das andere favoritedate (im Prinzip das Kaufdatum)

Sippl commented on app/src/main/java/de/projektss17/bonpix/daten/C_Favoriten_Adapter.java:
```diff
@@ -0,0 +1,108 @@
+            super(view);
+            icon = (ImageView) view.findViewById(R.id.favoriten_view_laden_bild);
+            favoriteShopName = (TextView) view.findViewById(R.id.favoriten_view_favoritenbeginn);
+            favoriteDate = (TextView) view.findViewById(R.id.favoriten_view_zusatz_favoritenende);
```
hier favoritenende

Sippl commented on app/src/main/res/layout/box_favoriten_view.xml:
```diff
@@ -0,0 +1,53 @@
+        android:src="@drawable/common_google_signin_btn_icon_dark_focused" />
+
+    <TextView
+        android:id="@+id/favoriten_view_favoritenbeginn"
```
wie oben bereits erwähnt

Sippl commented on app/src/main/res/layout/box_favoriten_view.xml:
```diff
@@ -0,0 +1,53 @@
+        android:textSize="18sp" />
+
+    <TextView
+        android:id="@+id/favoriten_view_zusatz_favoritenende"
```
wie oben bereits erwähnt

Sippl reviewed (changes requested):
Hi Diddy, bitte noch die Änderungen berücksichtigen (hoffe die haben sich jetzt mit deiner neuen Änderung nicht überschnitten). :-)

ImSeDanny committed:
Namen angepasst

C_Favoriten_Adapter:
- favoritenbeginn --> favoriteShopName
- favoritenende --> favoriteDate

box_favoriten_view.xml:
- favoritenbeginn --> favoriteShopName
- favoritenende --> favoriteDate

A_Favoriten:
- unnötigen Import gelöscht (war noch von der A_Garantie drinnen)

schmitzdominic reviewed (approved):
Ich finds sehr gut. 
PS: Ich mag auch den Mülleimer 👍 
Die Kleinigkeiten hat ja der Fabi schon gefunden, von dem her gibts von mir ein Approve

Sippl reviewed (approved):
Ok grünes Licht
